content/en/blog/2023/native-sidecars/index.md

```diff
@@ -0,0 +1,201 @@
+    image: istio/base
+    command:
+    - curl
+    - httpbin.org/get
+  containers:
+  - name: sleep
+    image: istio/base
+    command: ["/bin/sleep", "infinity"]
+{{< /text >}}
+
+Checking the proxy container, we can see the request both succeeded and went through the Istio sidecar:
+
+{{< text shell >}}
+$ kubectl logs sleep -c istio-proxy | tail -n1
+[2023-07-25T22:00:45.703Z] "GET /get HTTP/1.1" 200 - via_upstream - "-" 0 1193 334 334 "-" "curl/7.81.0" "1854226d-41ec-445c-b542-9e43861b5331" "httpbin.org" ...
+{{< /text >}}
+
+### Exiting pods
+
+Earlier we mentioned that when applications exit (common in `Jobs`), the Pod would live forever.
+Fortunately, this is addressed as well!
+
+First we deploy a pod that will exit after one second and doesn't restart:
+
+{{< text yaml >}}
+apiVersion: v1
+kind: Pod
+metadata:
+  name: sleep
+spec:
+  restartPolicy: Never
+  containers:
+- name: sleep
+  image: istio/base
+  command: ["/bin/sleep", "1"]
+{{< /text >}}
+
+And we can watch its progress:
+
+{{< text shell >}}
+$ kubectl get pods -w
+NAME    READY   STATUS     RESTARTS   AGE
+sleep   0/2     Init:1/2   0          2s
+sleep   0/2     PodInitializing   0          2s
+sleep   1/2     PodInitializing   0          3s
+sleep   2/2     Running           0          4s
+sleep   1/2     Completed         0          5s
+sleep   0/2     Completed         0          12s
+{{< /text >}}
+
+Here we can see the application container exited, and shortly after Istio's sidecar container exits as well.
+No more zombie pods!
+
+## What about ambient mode?
+
+Last year, Istio announced [ambient mode](/blog/2022/introducing-ambient-mesh/) - a new data plane mode for Istio that doesn't rely on sidecar containers.
+So with that coming, does any of this even matter?
+
+I would say a resounding "Yes"!
+
+While the impacts of sidecar are lessened when ambient mode is used for a workload, I expect that almost all large scale Kubernetes users have some sort of sidecar in their deployments.
+This could be Istio workloads they don't want to migrate to ambient, that they haven't *yet* migrated, or things unrelated to Istio.
+So while there may be a less scenarios that this matters, it still is a huge improvement for the cases where sidecars are used.
+
+You may wonder the opposite - if all our sidecar woes are addressed, why do we need ambient mode at all?
+There are still a variety of benefits ambient brings with these sidecar limitations addressed.
+For example, [this blog post](/blog/2023/waypoint-proxy-made-simple/) goes into details about why decoupling proxies from workloads is advantageous, and [this post](/blog/2023/ambient-performance-savings/) covers some of the cost saving opportunities.
+
+## Try it out yourself
+
+We encourage the adventurous readers to try this out themselves in testing environments!
+Feedback for these experimental and alpha features is critical to ensure they are stable and meeting expectations before shipping broader.
```

Comment:
we are especially looking for the feedback on 
- the shutdown sequence and multiple sidecars shutdown sequence.
- backoff timeout when envoy is crashing. Anything we can do wrt readiness of a pod or liveness of a Pod when sidecars present
- any other scenarios we forgot

Reply:
would be nice it you can list the feedback items explicitly